Infrastructure.DataAccess/Migrations/202209301046217_RemovedEmailBeforeDeletion.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Infrastructure.DataAccess.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class RemovedEmailBeforeDeletion : DbMigration
+    {
+        public override void Up()
+        {
+            DropColumn("dbo.User", "EmailBeforeDeletion");
+            Sql(@"
+                UPDATE [User]
```

Comment:
remember to test this migration by upgrading a prod backup db instance (remember to mark some users as deleted there)